nextflow.config

```diff
@@ -144,6 +144,7 @@ profiles {
     shifter.enabled = false
     charliecloud.enabled = false
     singularity.autoMounts = true
+    singularity.runOptions = "--bind \${NXF_WORK:-$launchDir/work}/GEMmaker"
```

Comment:
We don't need this type of setting on Kamiak and GEMmaker works just fine without this option.  If this is a Palmetto specific tweak then it needs to go into a local `nextflow.config` file rather than in the GEMmaker config file.  If you want to create a Clemson Institutional profile then you can put this there as well.  https://nf-co.re/usage/tutorials/step_by_step_institutional_profile